updating/updating_a_cluster/control-plane-only-update.adoc

```diff
@@ -14,53 +14,53 @@ To do: Remove this comment once 4.13 docs are EOL.
 
-// EUS-to-EUS update
-include::modules/updating-eus-to-eus-upgrade.adoc[leveloffset=+1]
+// Control Plane Only update
+include::modules/updating-control-plane-only-update.adoc[leveloffset=+1]
 
-// EUS-to-EUS update using the web console
-include::modules/updating-eus-to-eus-upgrade-console.adoc[leveloffset=+2]
+// Control Plane Only update using the web console
+include::modules/updating-control-plane-only-update-console.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
-[id="additional-resources_updating-eus-to-eus-upgrade-console"]
+[id="additional-resources_updating-control-plane-only-upgrade-console"]
 .Additional resources
 
 * xref:../../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Updating installed Operators]
 * xref:../../updating/updating_a_cluster/updating-cluster-web-console.adoc#update-upgrading-web_updating-cluster-web-console[Updating a cluster by using the web console]
 * xref:../../updating/updating_a_cluster/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating a cluster that includes RHEL compute machines]
 
-// EUS-to-EUS update using the CLI
-include::modules/updating-eus-to-eus-upgrade-cli.adoc[leveloffset=+2]
+// Control Plane Only update using the CLI
+include::modules/updating-control-plane-only-update-cli.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
-[id="additional-resources_updating-eus-to-eus-upgrade-cli"]
+[id="additional-resources_updating-control-plane-only-upgrade-cli"]
 .Additional resources
 
 * xref:../../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Updating installed Operators]
 * xref:../../updating/updating_a_cluster/updating-cluster-rhel-compute.adoc#updating-cluster-rhel-compute[Updating a cluster that includes RHEL compute machines]
 
-include::modules/updating-eus-to-eus-layered-products.adoc[leveloffset=+2]
+include::modules/updating-control-plane-only-layered-products.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
-[id="additional-resources_updating-eus-to-eus-layered-products"]
+[id="additional-resources_updating-control-plane-only-layered-products"]
 .Additional resources
 
 * xref:../../operators/admin/olm-upgrading-operators.adoc#olm-upgrading-operators[Updating installed Operators]
-* xref:../../updating/updating_a_cluster/eus-eus-update.adoc#updating-eus-to-eus-upgrade-console_eus-to-eus-update[EUS-to-EUS update using the web console]
-* xref:../../updating/updating_a_cluster/eus-eus-update.adoc#updating-eus-to-eus-upgrade-cli_eus-to-eus-update[EUS-to-EUS update using the CLI]
+* xref:../../updating/updating_a_cluster/control-plane-only-update.adoc#updating-control-plane-only-upgrade-console_control-plane-only-update[Performing a Control Plane Only update using the web console]
+* xref:../../updating/updating_a_cluster/control-plane-only-update.adoc#updating-control-plane-only-upgrade-cli_control-plane-only-update[Performing a Control Plane Only update using the CLI]
```

Comment:
I think that the anchors in these xref paths still use the old IDs and that maybe that's why the build is breaking:
```suggestion
* xref:../../updating/updating_a_cluster/control-plane-only-update.adoc#updating-control-plane-only-update-console_control-plane-only-update[Performing a Control Plane Only update using the web console]
* xref:../../updating/updating_a_cluster/control-plane-only-update.adoc#updating-control-plane-only-update-cli_control-plane-only-update[Performing a Control Plane Only update using the CLI]
```